contrib/nnn/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "nnn"
+pkgver = "4.9"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+hostmakedepends = ["pkgconf", "gmake"]
+makedepends = ["musl-fts-devel", "readline-devel"]
```

Comment:
libedit-readline-devel should be preferred unless there's a reason for the otherwise

Reply:
it required a patch which I just added